samples/ImplicitFlow01/AppWithFetchClient/wwwroot/app.js

```diff
@@ -0,0 +1,72 @@
+/* global window, fetch, Oidc */
+
+//
+// Define helper functions.
+//
+const fetchUserResources = (accessToken) => {
+  const url = `${window.location.origin}/api`;
+  const options = {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  };
+
+  // See https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+  fetch(url, options)
+    .then(response => response.json())
```

Comment:
More poor mobile browser support, what's your mobile strategy?

Reply:
Good point. 

I am not considering mobile for this demo. For the purpose of this demo, which is for teaching developers how OpenID Connect works, having limited mobile support should be fine. I expect that developers will be running this demo only on their development machines. It will be their responsibility to adapt it to production code. 

That said, I took a look at GitHub's fetch polyfill and installed it. It helps but does not have a lot of mobile browser testing. 

For the sake of keeping the demo simple, I'm not going to do any fancy mobile browser support. If you have a simple solution that works in popular mobile and desktop browser, I would love to know.